### PR TITLE
SAE-J3016 - revision 2021 - logical descriptions for classes

### PR DESCRIPTION
**Pull request by mskulim**
Signed-off-by: mskulim <mar.skulim@gmail.com>

### CODE REVIEW

File: AV/VehicleAutomationLevels_2021.rdf [hunk 1]
```diff
@@ -662,6 +1401,31 @@
...
+					<owl:Class>
+						<owl:unionOf rdf:parseType="Collection">
+							<rdf:Description rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/FullDrivingAutomation">
+							</rdf:Description>
+							<rdf:Description rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/HighDrivingAutomation">
+							</rdf:Description>
+						</owl:unionOf>
+					</owl:Class>
+				</owl:someValuesFrom>
+			</owl:Restriction>
```
Comment (mereolog) [inline AV/VehicleAutomationLevels_2021.rdf:1427]: This assume that all passenger must be involved in full or high driving automations.

**Review by mereolog — CHANGES_REQUESTED** [review]

File: AV/VehicleAutomationLevels_2021.rdf [hunk 2]
```diff
@@ -571,6 +1219,46 @@
 	
 	<owl:Class rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/MotorVehicleOperation">
 		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/Operation"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/hasOnboardUser"/>
+				<owl:onClass rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/InVehicleDriver"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
```
[the remark this answers is not in the document]
Comment (mskulim) [reply AV/VehicleAutomationLevels_2021.rdf:1227]: A motor vehicle operation does not need to have any onboard user. For example, a Level 5 dual-mode vehicle can be dispatched in driverless operation to go to a designated place and pick-up (onboard) passengers.

File: AV/VehicleAutomationLevels_2021.rdf (hunk 1 above)
Comment (mskulim) [reply AV/VehicleAutomationLevels_2021.rdf:1427]: I propose the following change to the definition of Passenger class: 
'is realized in at some time' some **('full driving automation' or 'high driving automation')** --> 'is realized in at some time' some **'sustained operation of a vehicle'**

Comment (mereolog) [reply AV/VehicleAutomationLevels_2021.rdf:1427]: It makes sense - go ahead with this change.

File: AV/VehicleAutomationLevels_2021.rdf [hunk 3]
```diff
@@ -38,14 +38,26 @@
...
+	</owl:AnnotationProperty>
+	
 	<owl:AnnotationProperty rdf:about="&skos;comment">
 	</owl:AnnotationProperty>
 	
+	<owl:AnnotationProperty rdf:about="&skos;example">
+	</owl:AnnotationProperty>
+	
 	<owl:AnnotationProperty rdf:about="&skos;explanatoryNote">
 	</owl:AnnotationProperty>
```
Comment (mereolog) [inline AV/VehicleAutomationLevels_2021.rdf:51]: Delete these annotations.

**Review by mereolog — CHANGES_REQUESTED** [review]

**Review by mereolog — APPROVED** [review]

**Pull request merged**